wire-compiler/src/main/java/com/squareup/wire/schema/Target.kt

```
@@ -90,6 +94,10 @@ sealed class Target {
                 "to $outDirectory", e)
           }
         }
+
+        override fun handle(service: Service) {
+          throw IllegalStateException("Service handling isn't supporting in Java.")
```

silently fail instead please?